src/theme/sprinkles.css.ts

```diff
@@ -0,0 +1,59 @@
+import { createSprinkles, defineProperties } from "@vanilla-extract/sprinkles";
+import { vars } from "./contract.css";
+
+const responsiveProperties = defineProperties({
+  conditions: {
+    mobile: {},
+    tablet: { "@media": "screen and (min-width: 768px)" },
+    desktop: { "@media": "screen and (min-width: 1024px)" },
+  },
+  defaultCondition: "mobile",
+  properties: {
+    display: ["none", "flex"],
```

Comment:
Are we going to defined list of all supported option here? E.g. grid ?

Reply:
Yes - for now I've added `flex` 😄